lavanyagarg112/meetwise
# Update development to sync with minor frontend changes

## Pull Request Template

**Description:**

There are minor changes made to frontend - team members update, section bug fixed, active organisation updates accordingly

**Related Issue**
Closes #120 
Closes #122 

**Testing**

1. Go to About Us and see if the changes are reflected
2. Make a new organisation, and see if active organisation is automatically updated
3. Click on the surrounding white box around the sidebar elements, and check if it navigates

**Types of Changes:**

- Bug fix: section bug
- New feature: active org update
- Info update: team members list

**Checklist:**

* [x] I have read and followed the contributing guidelines for this project.
